charts/posthog/templates/events-deployment.yaml

```diff
@@ -115,6 +115,9 @@ spec:
 {{- end }}
 {{- if .Values.web.env }}
 {{ toYaml .Values.web.env | indent 8 }}
+{{- end }}
```

Comment:
In order to avoid key collisions, what about having `.Values.events.env` first and then fallback with an elsif `.Values.web.env` to the current default?